release/device-rest-go.yaml

```
@@ -0,0 +1,19 @@
+---
+name: 'device-rest-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '09739088bb3ccddf347dd8b212f45a4687ee065f'
```

```suggestion
commitId: '7759c41568adf15208c0dd5cadcdb8f87895acd8'
```
This is the final commit for the release.